source/features/new-repo-disable-projects-and-wikis.tsx

```diff
@@ -41,6 +41,7 @@ async function init(signal: AbortSignal): Promise<void> {
 	const anchor = lastElement([
 		'.js-repo-init-setting-container', // IsNewRepo
 		'.form-checkbox', // IsNewRepoTemplate
+		'form.container-md > div:nth-last-child(3)',
```

Comment:
A comment should be added. What is this? I assume
```suggestion
		'form.container-md > div:nth-last-child(3)', // IsNewRepo
```

And the previous one needs to be either removed or updated to: `// TODO: Remove in June 2024. IsNewRepo`